src/BlockchainReader/Controller.ts

```diff
@@ -64,19 +64,37 @@ export class Controller {
     const matchingAnchors = filter(anchorPrefixAndVersionMatchConfiguration, anchors)
     const unmatchingAnchors = reject(anchorPrefixAndVersionMatchConfiguration, anchors)
 
+    const previousBlockHash = block.previousblockhash
+
+    const previousHashByHeight = await this.dao.findHashByHeight(blockHeight - 1)
```

Comment:
I'm thinking `previousBlockHash` and `previousHashByHeight` can be confusing names. Maybe rename to something like `incomingBlockPreviousHash` and `ourPreviousHash`? 

Suggestions welcome :)

Reply:
I like `previousBlockHash` as we are referring to the incoming block as the `block`.
I think we can improve `previousHashByHeight` to either `ourPreviousHash` as you said or maybe`storedPreviousHash`?

Reply:
I'm trying to find any "official" or popular vocabulary for this but haven't found anything so far.

What do you think of `localPreviousHash`?

Reply:
that sounds good